front/src/applications/stdcmV2/components/StdcmWarningBox.tsx

```diff
@@ -24,7 +30,14 @@ const StdcmWarningBox = ({
       <span>
         <Alert variant="fill" />
```

Comment:
Too small compared to the mockup

```suggestion
        <Alert variant="fill" size="lg" />
```

Reply:
Done